module/zfs/zfs_ioctl.c

```diff
@@ -6232,6 +6233,7 @@ static const zfs_ioc_key_t zfs_keys_send_new[] = {
 	{"embedok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
 	{"compressok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
 	{"rawok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
+	{"partialok",		DATA_TYPE_BOOLEAN,	ZK_OPTIONAL},
```

Comment:
Please update the comment above as well.

If you run `zfs send --partial` on a system with new userland and old kernel, does this ensure we get a reasonable error message?

Reply:
comment updated

yes, we get: `warning: cannot send 'data/test': the loaded zfs module does not support an option for this operation. A reboot may be required to enable this option.`